source/state_representation/include/state_representation/space/cartesian/CartesianState.hpp

```diff
@@ -549,7 +549,8 @@ inline const Eigen::Quaterniond& CartesianState::get_orientation() const {
 inline Eigen::Vector4d CartesianState::get_orientation_coefficients() const {
   return Eigen::Vector4d(
       this->get_orientation().w(), this->get_orientation().x(), this->get_orientation().y(),
-      this->get_orientation().z());
+      this->get_orientation().z()
+      );
```

Comment:
ok now this is weird. Did you do that manually? If yes then it should have one tab less in front of the `);`

Reply:
yes i did it manually. ok